dtschema/schemas/thermal/thermal.yaml

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: BSD-2-Clause
+# Copyright 2023 Collabora Ltd.
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/thermal/thermal.yaml#
+$schema: http://devicetree.org/meta-schemas/base.yaml#
+
+title: Thermal Consumer Common Properties
+
+maintainers:
+  - Rob Herring <robh@kernel.org>
+
+# always select the core schema
+select: true
+
+properties:
+  thermal-zones:
```

Comment:
Needs a description.

Reply:
Description added to both thermal-zones and thermal-zone-names